doc/source/building/windows.rst

```diff
@@ -168,90 +281,115 @@ the following in your build log:**
 
    FOUND:
       libraries = ['openblas']
-      library_dirs = ['C:\opt\64\lib']
-      language = c
+      library_dirs = ['C:\...........\lib']
+      language = f77
       define_macros = [('HAVE_CBLAS', None)]
 
+We'll get back to this below how to check.
```

Comment:
> **Before continuing, make sure that you don't have other copies of either**

Any suggestions other than searching C drive including subfolders? 
![image](https://user-images.githubusercontent.com/6570539/68091853-e16b3000-fe39-11e9-88af-97c53d6792bb.png)

Reply:
That line is not necessary I'll remove that part.

Reply:
So the search turned up:
![image](https://user-images.githubusercontent.com/6570539/68092237-ee8a1e00-fe3d-11e9-9e21-f22ed83cfd08.png)

I don't need to get rid of that file? It will not interfere?

Reply:
Hmm I have no idea. Then maybe it is the conda's MKL. Hence try to build SciPy directly. If it goes through then we are good.

Reply:
OK I will try. I built OpenBLAS first in case I need it.

Reply:
Also search turned up:
![image](https://user-images.githubusercontent.com/6570539/68097236-c1079980-fe6a-11e9-8be1-2be5442d4e81.png)